File: Beecrowd/1 - Beginner/1101 - 1190/1180 - Lowest Number and Position/1180_LowestNumberandPosition.py
number = int(input())

numbers = input().split()

lowest_number = int(numbers[0])
lowest_number_position = 1

for i in range(1, number):
    if int(numbers[i]) < lowest_number:
        lowest_number = int(numbers[i])
        lowest_number_position = i + 1

print(f"Menor valor: {lowest_number}")
print(f"Posicao: {lowest_number_position - 1}")

File: Beecrowd/1 - Beginner/1101 - 1190/1180 - Lowest Number and Position/1180_LowestNumberandPosition.c
#include <stdio.h>

int main() {
    int number;
    scanf("%d", &number);

    int lowestNumber = 0, lowestNumberPosition = 0;

    for (int i = 1; i <= number; i++) {
        int currentNumber;
        scanf("%d", &currentNumber);

        if (i == 1) {
            lowestNumber = currentNumber;
            lowestNumberPosition = i;
        } else {
            if (currentNumber < lowestNumber) {
                lowestNumber = currentNumber;
                lowestNumberPosition = i;
            }
        }
    }

    printf("Menor valor: %d\n", lowestNumber);
    printf("Posicao: %d\n", lowestNumberPosition - 1);
    
    return 0;
}
